main/lsp/TimeTravelingGlobalState.cc

```diff
@@ -118,6 +118,14 @@ vector<ast::ParsedFile> TimeTravelingGlobalState::indexFromFileSystem() {
         gs->errorQueue->drainWithQueryResponses();
     }
     globalStateHashes = computeStateHashes(gs->getFiles());
+
+    // When inputFileNames is 0 (as in tests), indexed ends up being size 0 because we don't index payload files.
+    // At the same time, we expect indexed to be the same size as GlobalStateHash, which _does_ have payload files.
+    // Resize the indexed array accordingly.
+    if (indexed.size() < gs->getFiles().size()) {
+        indexed.resize(gs->getFiles().size());
```

Comment:
Just noticing another `resize`-in-a-loop (not introduced in this PR, but adding one more `resize` here).

Is it possible to store the `pipeline::index` vector from line 109 in a local variable and loop over it once to find the max id so that it only resizes one time in this loop, rather than potentially resizing multiple times on line 112 when it's looping?  If so, could the resize also make the resize on line 126 unnecessary, or will there still have to be a resize after line 120?

Reply:
This statement isn't in a loop, so the resize only executes once.